auth/handler.go

```diff
@@ -28,30 +28,18 @@ func handleSpendMsg(ctx sdk.Context, uk db.UTXOKeeper, msg types.SpendMsg, txInd
 	utxo1 := uk.UM.GetUTXO(ctx, position1)
 	var position2 types.Position
 	var utxo2 types.UTXO
-	err := uk.SpendUTXO(ctx, msg.Owner1, position1)
-	if err != nil {
```

Comment:
Why take out the err check? How are `SpendUTXO` and `RecieveUTXO` guaranteed to not fail?

Reply:
The only error checks SpendUTXO and RecieveUTXO contained where for checking the existence of a utxo, which the ante handler does, but upon further thoughts I think the redundant checks should be added back. Are there any other errors that may occur other than if the bytes at the encoded position are nil?